main/stm32bl.c

```diff
@@ -86,11 +114,46 @@ int stm32bl_go(uint32_t addr)
     if (wait_resp() != STM32BL_ACK) {
         return ESP_FAIL;
     }
-
     send_address(addr);
     return wait_resp();
 }
 
+static int stm32bl_run_raw_code(uint32_t target_address, const uint8_t *code, uint32_t code_size)
+{
+	uint32_t stack_le = 0x20002000;
+	uint32_t code_address_le = target_address + 8 + 1; // thumb mode address (!)
+	uint32_t length = code_size + 8;
+	uint8_t *mem, *pos;
+	uint32_t address;
+
+	/* Must be 32-bit aligned */
+	if (target_address & 0x3) {
```

Comment:
Below I saw you used `% 4` to check if it is aligned. I suggest to do the same here as it is more obvious what it does and in order to make it consistent.